### PR TITLE
spark totalVolume onboarding

### PR DESCRIPTION
**Pull request by PaulZhemanov**

### CODE REVIEW

**Comment by llamatester** [comment]
The spark adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs spark
    
    🦙 Running SPARK adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 19 Nov 2024 00:00:00 GMT
    End Date:	Wed, 20 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    Trade Volume: 4794.2208423282755
    Timestamp: 1732093200
    FUEL 👇
    Backfill start time: 30/9/2020
    NO METHODOLOGY SPECIFIED
    Total volume: 4.79 k
    End timestamp: 1732093200 (2024-11-20T09:00:00.000Z)

**Comment by llamatester** [comment]
The spark adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs spark
    
    🦙 Running SPARK adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 19 Nov 2024 00:00:00 GMT
    End Date:	Wed, 20 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    Trade Volume: 4813.46517827553
    Timestamp: 1732111200
    FUEL 👇
    Backfill start time: 30/9/2020
    NO METHODOLOGY SPECIFIED
    Total volume: 4.81 k
    End timestamp: 1732111200 (2024-11-20T14:00:00.000Z)

**Comment by dtmkeng** [comment]
could you return dailyVolume ?

File: dexs/spark/index.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      }
+
+      const totalVolume = rows[0]?.tradeVolume;
+      const timestamp = rows[0]?.timestamp;
+
+      console.log('Trade Volume:', totalVolume);
+      console.log('Timestamp:', timestamp);
+
+      return {
+        totalVolume: totalVolume,
```
Comment (dtmkeng) [inline dexs/spark/index.ts]: could you add dailyVolume ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The spark adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs spark
    
    🦙 Running SPARK adapter 🦙
    ---------------------------------------------------
    Start Date:	Wed, 20 Nov 2024 00:00:00 GMT
    End Date:	Thu, 21 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    FUEL 👇
    Backfill start time: 30/9/2020
    NO METHODOLOGY SPECIFIED
    Daily volume: 55
    End timestamp: 1732182780 (2024-11-21T09:53:00.000Z)

**Comment by PaulZhemanov** [comment]
@llamatester Hi! could you check updates please?

**Comment by llamatester** [comment]
The spark adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs spark
    
    🦙 Running SPARK adapter 🦙
    ---------------------------------------------------
    Start Date:	Wed, 20 Nov 2024 08:20:55 GMT
    End Date:	Thu, 21 Nov 2024 08:20:55 GMT
    ---------------------------------------------------
    
    FUEL 👇
    Backfill start time: 6/11/2024
    NO METHODOLOGY SPECIFIED
    Daily volume: 54
    End timestamp: 1732177254 (2024-11-21T08:20:54.000Z)

**Comment by g1nt0ki** [comment]
@PaulZhemanov thanks for the PR

**Pull request merged**